README.md

```diff
@@ -53,7 +55,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+I beleive using this the memory requirment would be O(1) just becasue it will continue to just use two as it recursivly calls until the base case. 
```

Comment:
# -3

It has to push half of the input onto the stack in order to match the other half in reverse order, so it is O(n)